pom.xml

```diff
@@ -93,9 +96,27 @@
 
     <dependencies>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
Why does this need to be in the top level pom and not in the test module?

Reply:
Because using `test` module is a bad practice, unit tests should be in their modules. Right now we have a number of unit tests outside of `test` module

Reply:
I see. Why is it bad practice ?

Reply:
Unit tests inside their modules - is a recommended project structure for any standard Java project. Any build tool uses that agreement by default. 

Modularity suffers

You can't run tests for a particular module, like `./mvnw -pl infrastructure test` you can only run unit tests for all projects.

Unit tests and Integration tests sitting in the same project, if you want to run them separately you need to have custom commands for that (see our .travis.yml)